source/assets/demo/css/no-prod.css

```diff
@@ -114,47 +130,52 @@ th {
 }
 
 @media (max-width: 719px) {
-  th {
+  body.demo-doc-page th {
     font-size: 16px;
   }
 
-  thead th:last-of-type,
-  tbody td:last-of-type {
+  body.demo-doc-page thead th:last-of-type,
+  body.demo-doc-page tbody td:last-of-type {
     border-right-width: 8px;
   }
 }
 
-// if there is a thead, begin with a light bg-color
-thead + tbody tr:nth-child(even) {background-color: rgba(245,245,242,1)}
-thead + tbody tr:nth-child(odd) {background-color: rgba(250,250,248,1)}
+/* if there is a thead, begin with a light bg-color */
+body.demo-doc-page thead + tbody tr:nth-child(even) {background-color: rgba(245,245,242,1)}
```

Comment:
nix dogfooding? :)

Reply:
well, I know. the old html version of the docs used fef components. but unfortunately, we would have to put a lot of html just for the fef-css-classes into the .md-files. which kind of defeats the purpose of using md-files... ;-)